modules/auxiliary/scanner/http/onion_omega2_login.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+# 2019-03-27 05-55
+
+# Standard Modules
+from metasploit import module, login_scanner
+import json
+
+# Extra Modules
+dependencies_missing = False
+try:
+    import requests
+except ImportError:
+    dependencies_missing = True
+
+# Metasploit Metadata
+metadata = {
+    'name': 'Onion Omega2 Login Brute-Force',
+    'description': '''
+        OnionOS login scanner module for Onion Omega2 devices.
+    ''',
+    'authors': [
+        'Not So Attractive <github.com/nsa>'
+    ],
+    'date': '2019-03-27',
+    'license': 'MSF_LICENSE',
+    'references': [
+    ],
+    'type': 'single_host_login_scanner',
+    'options': {
+        'rhost': {'type': 'address', 'description': 'Host to target', 'required': True, 'default': '192.168.3.1'},
```

Comment:
Remove default rhost. I don't recall if default can be completed removed here.
```suggestion
        'rhost': {'type': 'address', 'description': 'Host to target', 'required': True},
```